lib/ts/recipe/passwordless/smsdelivery/services/twilio/index.ts

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+    ServiceInterface,
+    TypeInput,
+    normaliseUserInputConfig,
+    TypeNormalisedInput,
+} from "../../../../../ingredients/smsdelivery/services/twilio";
+import { SmsDeliveryInterface } from "../../../../../ingredients/smsdelivery/types";
+import * as Twilio from "twilio";
+import OverrideableBuilder from "supertokens-js-override";
+import { TypePasswordlessSmsDeliveryInput } from "../../../types";
+import { getServiceImplementation } from "./serviceImplementation";
+
+export default class TwilioService implements SmsDeliveryInterface<TypePasswordlessSmsDeliveryInput> {
+    serviceImpl: ServiceInterface<TypePasswordlessSmsDeliveryInput>;
+    private config: TypeNormalisedInput<TypePasswordlessSmsDeliveryInput>;
+
+    constructor(config: TypeInput<TypePasswordlessSmsDeliveryInput>) {
+        this.config = normaliseUserInputConfig(config);
+        const twilioClient = Twilio(
+            config.twilioSettings.accountSid,
+            config.twilioSettings.authToken,
+            config.twilioSettings.opts
+        );
```

Comment:
I'm thinking if we should just let the user pass the twilio client, instead of passing the configs for the twilio client. Won't it make it simpler for us..?

Reply:
@bhumilsarvaiya said that people are more familiar with passing in the authToken etc.. So resolving this